relay-metrics/src/aggregation.rs

```diff
@@ -641,7 +645,7 @@ fn tags_cost(tags: &BTreeMap<String, String>) -> usize {
 #[fail(display = "failed to parse metric bucket")]
 pub struct ParseBucketError(#[cause] serde_json::Error);
 
-/// An aggregation of metric values by the [`Aggregator`].
+/// An aggregation of metric values by the [`AggregatorService`].
```

Comment:
I would keep linking to the `Aggregator` interface in this and similar doc comments (Bucket etc), since the interface describes the opaque behavior for metrics aggregation.